### PR TITLE
Added schema: to generated files

### PR DESCRIPTION
**Pull request by THAlpha**

### CODE REVIEW

**Comment by THAlpha** [comment]
Any update on getting this reviewed/merged?

File: lib/index.js [hunk 1]
```diff
@@ -212,7 +212,10 @@ AutoSequelize.prototype.run = function(callback) {
             }
             else if (foreignKey.isForeignKey) {
               text[table] += spaces + spaces + spaces + "references: {\n";
-              text[table] += spaces + spaces + spaces + spaces + "model: \'" + self.tables[table][field][attr].foreignSources.target_table + "\',\n"
+              text[table] += spaces + spaces + spaces + spaces + "model: {\n"
+              text[table] += spaces + spaces + spaces + spaces + spaces + "tableName: \'" + self.tables[table][field][attr].foreignSources.target_table + "\',\n"
+              text[table] += spaces + spaces + spaces + spaces + spaces + "schema: \'" + self.tables[table][field][attr].foreignSources.target_schema + "\'\n"
```
Comment (durango) [inline lib/index.js]: Shouldn't this check for `options.schema`?

**Comment by THAlpha** [comment]
@durango Updated as per review

**Comment by steveschmitt** [comment]
Incorporated in sequelize-auto@0.5.0
Thanks!

**Pull request closed without merging**